packages/ddex/ingester/parser/parser.go

```diff
@@ -61,6 +72,93 @@ func RunNewParser(ctx context.Context) {
 	}
 }
 
+// createArtistNameIndex creates an index on the name (display name) field in the 'users' collection
+func (p *Parser) createArtistNameIndex() error {
+	_, err := p.usersColl.Indexes().CreateOne(p.ctx, mongo.IndexModel{
```

Comment:
i assume this does nothing if the index already exists?

Reply:
yep! should basically be a "create if not exists"